server/src/main/java/org/elasticsearch/action/admin/cluster/stats/AnalysisStats.java

```diff
@@ -176,6 +196,28 @@ private static void aggregateAnalysisTypes(
         }
     }
 
+    private static void aggregateSynonymFilterTypes(
+        Collection<Settings> filterSettings,
+        Map<String, IndexFeatureStats> stats,
+        String synonymType,
+        String indexName,
+        Set<String> indicesAdded
+    ) {
+        for (Settings filterComponentSettings : filterSettings) {
+            final String type = filterComponentSettings.get("type");
+            if ((type != null) && type.startsWith("synonym")) {
```

Comment:
I think it is better to be specific here: `type.equals("synonym") || type.equals("synonym_filter")`, as plugins can introduce their own filters with any name they like.

Reply:
Understood, thanks!